### PR TITLE
Update 5.46

### PR DESCRIPTION
**Pull request by blmaier**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -77,7 +85,7 @@ outputs:
 about:
   home: http://darwinsys.com/file
   summary: Fine Free File Command
-  license: BSD-2-Clause
+  license: BSD-2-Clause-Darwin
```
Comment (blmaier) [inline recipe/meta.yaml:83]: Note this is the correct SPDX license identifier. It's a identifier specifically for `file`.
https://spdx.org/licenses/BSD-2-Clause-Darwin.html

The linter warns about it as it appears to be more recently added to SPDX. I opened a PR to smithy to update the SPDX license list.
https://github.com/conda-forge/conda-smithy/pull/2177

**Comment by blmaier** [comment]
@chrisburr @mariusvniekerk @mrakitin could I get a review on this?

**Comment by blmaier** [comment]
Hi @conda-forge/core, this repo appears to be abandoned, could you look at it?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -77,11 +80,12 @@ outputs:
...
   license_file: COPYING
   dev_url: https://github.com/file/file
 
 extra:
+  feedstock-name: {{ name|lower }}
   recipe-maintainers:
```
Comment (h-vetinari) [inline recipe/meta.yaml:89]: Please add yourself to maintainers and rerender

**Review by h-vetinari — COMMENTED** [review]
Thanks for reviving this feedstock!

**Review by h-vetinari — APPROVED** [review]

**Pull request merged**